src/ui-config/markets/index.ts

```diff
@@ -62,6 +85,162 @@ export const marketsData: { [key in keyof typeof CustomMarket]: MarketDataType }
       },
     },
     cornAirdropAddress: '0xae763973B873718e084a2ea3F7aE50B0c670bdF0',
+    nearConfig: {
+      networkId: 'mainnet',
+      keyStore: new keyStores.BrowserLocalStorageKeyStore(),
+      nodeUrl: 'https://rpc.mainnet.near.org',
+      walletUrl: 'https://wallet.mainnet.near.org',
+      helperUrl: 'https://helper.mainnet.near.org',
+      headers: {},
+    },
+    cornerstoneSDKConfig: {
```

Comment:
see the source code, I found the function in this project use these params.

` this.config.refPoolForCorn`
` this.config.refContractId`
`this.config.cornTokenDecimals`

```
 cornerstoneSDKConfig: {
      cornerstoneConfig: {
        cornContractId: 'corn.v1.corn-staging.near',
        xcornContractId: 'xcorn.v1.corn-staging.near',
        vecornContractId: 'vecorn.v1.corn-staging.near',
      },
      dataServiceConfig: {
        cornTokenDecimals: 18,
        xcornTokenDecimals: 18,
        vecornTokenDecimals: 18,
        refContractId: 'v2.ref-finance.near',
        refPoolForXcorn: {
          poolId: 3709,
          otherTokenId: 'usdc.mocks.near',
          otherTokenDecimals: 6,
        },
        cornOracleContractId: 'lpt.corn-oracle.near',
        fluxOracleContractId: 'fpo.opfilabs.near',
        fluxPairInfos: {},
        coinGeckoIds: {},
        treasuryWhiteListTokens: {
          ft: [],
          mft: [],
        },
      }
```

keep these config above. because the config params are not optional type

Reply:
sounds good